README.org

```diff
@@ -37,31 +37,31 @@ For org-static-blog, a blog consists of six parts:
   lists all other posts of the same tag. Additionally, a tag overview
   page is created that lists the publishing dates and headlines of
   every blog post, sorted by tags. This feature is only enabled if you
-  set `org-static-blog-enable-tags` to `t`.
+  set =org-static-blog-enable-tags= to =t=.
 
 Every HTML page in org-static-blog can be customized in the following
 ways:
-- The contents of `org-static-blog-page-header` are inserted into the
-  `<head>` of every page. Use this to include custom CSS and
+- The contents of =org-static-blog-page-header= are inserted into the
+  =<head>= of every page. Use this to include custom CSS and
   JavaScript for your blog.
-- The contents of `org-static-blog-page-preamble` is inserted just
+- The contents of =org-static-blog-page-preamble= is inserted just
   before the content of every page. This is a good place to put the
   header or menus for your blog.
-- The contents of `org-static-blog-page-postamble` is inserted just
+- The contents of =org-static-blog-page-postamble= is inserted just
   after the content of every blog entry, but not in the index page and
   the archive. This is where you can include copyright notices or
   comment boxes.
-- The return values of `org-static-blog-post-preamble` and
-  `org-static-blog-post-postamble` are prepended and appended to every
+- The return values of =org-static-blog-post-preamble= and
+  =org-static-blog-post-postamble= are prepended and appended to every
   blog post. If you want to change the formatting of dates, titles, or
   the tag list, overwrite these functions.
 
 If you want to activate a few convenience key bindings, add
-`(add-to-list 'auto-mode-alist (cons (concat org-static-blog-posts-directory ".*\\.org\\'") 'org-static-blog-mode))`
-to your *init.el*. These key bindings are:
-- `C-c C-f`/`C-c C-b` to open next/previous post.
```

Comment:
this line does not render correctly.

Reply:
Hmm... I checked source and how it's rendered by GitHub - looks the same for me?...

Reply:
It renders as "C-c C-f=/=C-c C-b" (note the equal signs)

Reply:
Oh, sorry. Fixed :)